src/app/app/organizations/edit/edit-organization.component.ts

```diff
@@ -20,12 +20,14 @@ export class EditOrganizationComponent implements OnInit {
   // Data
   id: number = 0;
   organization: Organization = new Organization();
+  slackTeam: SlackTeam = new SlackTeam();
   authorities: Map<string, OrganizationAuthorityModel> = new Map<string, OrganizationAuthorityModel>()
 
   // Forms
   editOrgForm: FormGroup = this.formBuilder.group({
     name: [this.organization.name, Validators.required],
-    logoUrl: [this.organization.logoUrl]
+    logoUrl: [this.organization.logoUrl],
+    slackPublicationChannelId: [this.slackTeam.publicationChannelId]
```

Comment:
Be careful : the value you use to initialize the field is the default value ID, not the current one. 

It is because at this point you did not retrieve the content of slackTeam (which will be stored in `this.organization.slackTeam` in the `refreshSlackTeam()` method.

Reply:
je viens de corriger pour prendre this.organization.slackTeam.publicationChannelId mais ça n'affiche pas la valeur. 
J'ai regardé pour le name ( this.organisation.name) et tu fais : 
 this.editOrgForm.controls['name'].setValue(this.organization.name);
 
 La valeur dans le FormGroup n'est pas sensé servir à ça ?

Reply:
mais au lieu d'utiliser setValue il vaut mieux mettre sur le champ directement :
[value]="organization.slackTeam.publicationChannelId"

<input type="text" formControlName="slackPublicationChannelId" [value]="organization.slackTeam.publicationChannelId" class="form-control" id="slackPublicationChannelId" placeholder="Enter channel id for slack">

Reply:
pareil pour l'url du logo qui n'est pas remonté du coup pour le moment

Reply:
La valeur qu'on met dans FormGroup est celle à l'initialisation sauf que l'attribut `slackTeam` ne correspond pas à `organization.slackTeam`. La valeur n'est pas rafraîchie automatiquement c'est pour ça que j'utilise la fonction `.setValue()`.

Ce qui est intéressant en utilisant la valeur de FormGroup c'est qu'on peut y mettre des contraintes de validation (nombre de caractères, format de la chaîne, etc.) ce que la simple définition de ngModel ne permet pas.

Le logo n'est pas remonté non plus en effet. Tu peux voir pour corriger ça en même temps stp ?